src/bindings/interface/annotator.i

```diff
@@ -0,0 +1,217 @@
+
+%feature("docstring") libcellml::Annotator::item
+"Return the item with the given id.";
+
+%feature("docstring") libcellml::Annotator::component
+"Return the component with the given id.";
+
+%feature("docstring") libcellml::Annotator::encapsulation
+"Return the model with the given encapsulation id.";
+
+%feature("docstring") libcellml::Annotator::model
+"Return the model assigned to this Annotator or the model with the given id.";
+
+%feature("docstring") libcellml::Annotator::variable
+"Return the variable with the given id.";
+
+%feature("docstring") libcellml::Annotator::importSource
+"Return the import source with the given id.";
+
+%feature("docstring") libcellml::Annotator::units
+"Return the units with the given id.";
+
+%feature("docstring") libcellml::Annotator::reset
+"Return the reset with the given id.";
+
+%feature("docstring") libcellml::Annotator::testValue
+"Return the ResetPtr whose test_value has the given id.";
+
+%feature("docstring") libcellml::Annotator::resetValue
+"Return the ResetPtr whose reset_value has the given id.";
+
+%feature("docstring") libcellml::Annotator::componentRef
+"Return the ComponentPtr with the given component_ref id.";
+
+%feature("docstring") libcellml::Annotator::unit
+"Return the UnitsPtr and index whose Unit is with given id.";
+
+%feature("docstring") libcellml::Annotator::connection
+"Return a pair of VariablePtrs that define a connection with given id.";
+
+%feature("docstring") libcellml::Annotator::mapVariables
+"Return a pair of VariablePtrs that define a map_variables with given id.";
+
+%feature("docstring") libcellml::Annotator::assignAllIds
+"Traverse the stored model and set any blank id fields to an automatically generated id.";
+
+%feature("docstring") libcellml::Annotator::assignIds
+"Set all items of the given type in the stored model to automatically generated, unique strings.";
+
+%feature("docstring") libcellml::Annotator::assignId
+"Set the id of the given item to an automatically generated, unique string.";
+
+%feature("docstring") libcellml::Annotator::clearAllIds
+"Clear all the id strings in the given or stored model.";
+
+%feature("docstring") libcellml::Annotator::isUnique
+"Return `true` if the given string is not duplicated in the stored model, or `false` otherwise.";
+
+%feature("docstring") libcellml::Annotator::ids
+"Return a list of all id strings within the stored model.";
+
+%feature("docstring") libcellml::Annotator::duplicateIds
+"Return a list of id strings which are duplicated within the stored model.";
+
+%feature("docstring") libcellml::Annotator::assignComponentId
+"Assign an automatically generated, unique id to the given component.";
+
+%feature("docstring") libcellml::Annotator::assignComponentRefId
+"Assign an automatically generated, unique id to the given component's encapsulation.";
+
+%feature("docstring") libcellml::Annotator::assignConnectionId
+"Assign an automatically generated, unique id to the connection between the given variables.";
```

Comment:
"Assign an automatically generated, unique id to the connection between the parent components of the given variables.";